### ISSUE TITLE
Example links should point to correct version

### ISSUE THREAD

**Issue by yawaramin** [aantron/dream#325]
On the Dream web page: https://aantron.github.io/dream/

All the example links seem to point at the `master` branch, eg https://aantron.github.io/dream/#sql

> Dream provides thin convenience functions over [Caqti](https://github.com/paurkedal/ocaml-caqti/#readme), an SQL interface with several back ends. See example [h-sql](https://github.com/aantron/dream/tree/master/example/h-sql#files)

However these examples can change significantly between the last released Dream version and the `master` branch. So they can fail to compile when the user tries them.

We should make sure all the links point to the tagged versions in the repo which correspond to the displayed latest opam release, eg: https://github.com/aantron/dream/tree/1.0.0-alpha5/example/h-sql

**Comment by aantron** [comment]
Agreed in principle. There are a few difficulties with maintaining synchronization between the examples and the latest release:

1. I tend to develop the examples together with `master`.
2. It takes some effort to maintain the right link to the latest release. Maybe we can use a moving ref for this purpose.

**Comment by aantron** [comment]
I think the best solution is to just get back to a regular release schedule, so that the amount of time the examples are out of sync with `master` is minimized. I've [proposed](https://github.com/ocaml/opam-repository/pull/25949) a release to opam-repository.
